libraries/chaindb/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Defines ChainBase library target.
+project( ChainBase )
+
+file(GLOB HEADERS "include/chainbase/*.hpp")
+add_library( chainbase src/chainbase.cpp src/pinnable_mapped_file.cpp ${HEADERS} )
+target_link_libraries( chainbase PUBLIC pthread Boost::system )
+
+if(TARGET Boost::asio)
+  target_link_libraries( chainbase PUBLIC
+                         Boost::headers Boost::interprocess Boost::chrono Boost::multi_index Boost::lexical_cast Boost::asio )
+endif()
+
+target_include_directories( chainbase PUBLIC
+  "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+  "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"  )
+
+enable_testing()
+add_subdirectory( test )
+add_subdirectory( benchmark )
+
+if(CHAINBASE_INSTALL_COMPONENT)
+   set(INSTALL_COMPONENT_ARGS COMPONENT ${CHAINBASE_INSTALL_COMPONENT} EXCLUDE_FROM_ALL)
+endif()
```

Comment:
You can remove the if and setting of `INSTALL_COMPONENT_ARGS`, and then hardcode `COMPONENT dev` like we do for fc,
https://github.com/AntelopeIO/spring/blob/504fa82dca9172a36056a67dc1932e063bdb6adf/libraries/libfc/CMakeLists.txt#L111-L114
I did notice that failed to remove line 2 here when doing fc way back. If you make this change it is fine to remove line 1 and 2
https://github.com/AntelopeIO/spring/blob/504fa82dca9172a36056a67dc1932e063bdb6adf/libraries/CMakeLists.txt#L1-L2

Reply:
https://github.com/AntelopeIO/spring/pull/1449/commits/31a08eee8c7b41ebad9bdd39bfcb9e7380c277f3